lib/google_maps_geocoder/google_maps_geocoder_error_handler.rb

```diff
@@ -0,0 +1,17 @@
+# Handling errors form Google
+module GoogleMapsGeocoderErrorHandler
+  # Parent class for more concrete errors
+  class GeneralGoecodingError < StandardError
+    def initialize(response_json = '')
+      @json = response_json
+      super
+    end
+
+    def message
+      "Google returned:\n#{@json.inspect}"
+    end
+  end
```

Comment:
yeah, I renamed this and got rid of the module as well, please see second commit